tests/gold_tests/tls/tls_client_versions.test.py

```diff
@@ -41,9 +41,7 @@
 ts.addSSLfile("ssl/server.pem")
 ts.addSSLfile("ssl/server.key")
 
-ts.Variables.ssl_port = 4443
-
-# Need no remap rules.  Everything should be proccessed by sni
+# Need no remap rules.  Everything should be proccessed by ssl_server_name
```

Comment:
this should be sni

Reply:
changed all the instances of ssl_server_name to sni